<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductoMenu</class>
 <widget class="QWidget" name="ProductoMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>160</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>160</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QSpinBox" name="cantidadSB"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="agregarPB">
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="nombreLB">
      <property name="text">
       <string>nombre</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="precioLB">
      <property name="text">
       <string>precio</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="imagenLB">
      <property name="minimumSize">
       <size>
        <width>139</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>139</width>
        <height>120</height>
       </size>
      </property>
      <property name="text">
       <string>IMAGEN</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
